.github/workflows/lint.yml

```diff
@@ -22,7 +22,7 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3.1.0
         with:
           fetch-depth: 0
       - uses: actions/setup-python@v4
```

Comment:
Forgot to pin the `setup-python@v4`

Reply:
Oops, messed up my vscode search/replace, thanks @sogustavo.